src/components/common/MessagePopup.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PopupWrapper from './PopupWrapper';
+import Button from './Button';
+import styled from 'styled-components';
+
+interface MessagePopupProps {
+  isOpen: boolean;
+  onClose: () => void;
+  message: string;
+  title: string;
+  type?: 'info' | 'warning' | 'error';
```

Comment:
Optional request.
extract to separate type 
```suggestion
  type?: PopupType; 
```
type PopupType = 'info' | 'warning' | 'error';

Reply:
or maybe enum